codeception.yml

```diff
@@ -0,0 +1,16 @@
+namespace: tests\Functional\_support
+
+suites:
+    functional:
+        actor: FunctionalTester
+        path: .
+        modules:
+            enabled:
+                - PhpBrowser:
+                    url: http://nginx
+                - \Tests\Functional\_support\Helper\Functional
```

Comment:
What is the purpose of this line ?

Reply:
It doesn't have an use for now

Reply:
If your PR is not final, please add a WIP to tje PR title.